lib/howl/commands/file_commands.moon

```diff
@@ -39,6 +39,31 @@ command.register
   input: -> interact.select_file allow_new: true
   handler: (file) -> app\open_file file
 
+command.register
```

Comment:
Hmm, it's not just this - maybe the last few commits didn't make it in, including the OSX binding and implementation cleanup.